.github/workflows/ci.yml

```diff
@@ -9,18 +9,15 @@ on:
 jobs:
   tests:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node: [16]
-        npm: [8.5.x]
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
+    - name: Setup Nodejs Env
+      run: echo "NODE_VER=`cat .nvmrc`" >> $GITHUB_ENV
     - name: Setup Nodejs
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v3
       with:
-        node-version: ${{ matrix.node }}
-    - run: npm install -g npm@${{ matrix.npm }}
+        node-version: ${{ env.NODE_VER }}
```

Comment:
Just curious if the `node-version-file` attribute was considered when constructing the workflow. We could retain the original strategy matrix attributes, but point directly to the `.nvmrc` file as a reference to the intended node version to upgrade to.
Docs: https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file

```yml
strategy:
    matrix:
        node-version-file: ['.nvmrc']     
...
steps:
    ...
    - name: Setup Nodejs Env version ${{ matrix.node-version-file}}
      uses: actions/setup-node@v3
      with:
          node-version-file: ${{matrix.node-version-file}}
```

Reply:
The workflow was created by arbi-bom originally and with node v18 upgrades it spread across all the remaining MFEs where it was not being used previously. I'm not sure if `node-version-file` was considered while constructing it but I could see the merit to retain the original strategy matrix attributes. We can explore this further, thanks for bringing it to our attention.